src/features/analytics/components/IndicatorComponent/SingleComponent.tsx

```diff
@@ -0,0 +1,12 @@
+import { IndicatorType } from '../../types';
+
+function SingleComponent({ value, icon }:{ value: IndicatorType, icon?: string }) {
+  return (
+    <div className={`single-component ${icon || ''}`}>
```

Comment:
please read this article (https://habr.com/ru/post/649381/) and fix the rendering of classes